concourse/util.py

```diff
@@ -32,13 +32,18 @@
 
 def list_github_resources(
     concourse_url: str,
+    concourse_version: int,
     concourse_user: str='kubernetes',
     concourse_passwd: str='kubernetes',
     concourse_team: str='kubernetes',
     concourse_pipelines=None,
     github_url: str=None,
 ):
-    concourse_api = concourse.ConcourseApi(base_url=concourse_url, team_name=concourse_team)
+    concourse_api = concourse.ConcourseApi(
+        base_url=concourse_url,
+        team_name=concourse_team,
+        concourse_version=concourse_version,
+    )
     concourse_api.login(
```

Comment:
btw - considering we basically always need to do a login as a first operation: why not do the login in the factory function already? this would remove a lot of redundant code